fishtank-cli/src/commands/mineuntil.ts

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CurrencyUtils } from '@ironfish/sdk'
+import { Command, Config, Flags } from '@oclif/core'
+import { Cluster } from 'fishtank'
+
+export abstract class MineUntil extends Command {
+  static description = 'Mine on the node until a condition is met'
+
+  static flags = {
+    nodeName: Flags.string({
+      char: 'n',
+      description: 'Bootstrap node name',
```

Comment:
Nit: this can be any node name, not necessarily a bootstrap node

Reply:
Copy paste, fixed